### PR TITLE
fixed uneven spacing between Date and category

### PR DESCRIPTION
**Pull request by sandy-fyle**
Before:

Screenshot (1)

After:

Screenshot (2)

Error(before):
Screenshot (5)

Error(after):
Screenshot (9)

### CODE REVIEW

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss [hunk 1]
```diff
@@ -271,6 +271,7 @@
   &--project-block {
     flex-grow: 1;
     margin-right: 26px;
+    margin-top: -13px;
```
Comment (Dimple16) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.scss:274]: check if `margin-top: -12px;` works

Comment (Dimple16) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.scss:274]: also add screenshots with error messages

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss [hunk 2]
```diff
@@ -605,7 +606,6 @@
   &--error {
     color: $red;
     line-height: 16px;
-    margin-top: -12px;
```
Comment (Dimple16) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.scss:608]: this will have an impact on all fields, please check

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss (hunk 2 above)
Comment (sandy-fyle) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.scss:608]: Hey date isn't mandatory now but it was required before so it was giving error and error of date was not aligned but now   I added margin-top: -12px back. Now it looks good to me.

e

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**